content/lessons/chapter-3/index.ts

```diff
@@ -1,11 +1,13 @@
 import * as Intro1 from './intro-1'
 import * as Intro2 from './intro-2'
 import * as SOL1 from './solo-1'
+import * as OUT1 from './outro-1'
```

Comment:
Nitpick but I'm not sure why we are using this different naming convention here.

Reply:
I missed the naming convention followed in other chapters while writing this part.
Thanks for catching it!

Addressed in 01cf4fc113094935cbe2b2dec8741ef5c82409e3